### DISCUSSION TITLE
Translation Method for built-in title fields

### DISCUSSION THREAD

**Discussion by brandonkelly** [craftcms/cms#8467] [Ideas]
Entry and category titles should have their own Translation Method settings on multi-site installs.

**Comment by jsunsawyer** [comment]
If this does ever get addressed, it'd be great to also have translation methods for slugs.

**Comment by kruncher** [comment]
Agreed with translation methods for slugs.

It would be nice if a custom translation method could be defined globally; perhaps by a plugin, and then if we could select "My Special Method" from the drop-down.

**Comment by benface** [comment]
In the meantime, a little improvement would be to clarify what the current translation method is when hovering the "translatable" icon next to the Title field. Right now it only says "This field is translatable", but I believe it should say "This field is translated for each site".

**Comment by brandonkelly** [comment]
@benface Yep. Done for the next release.

**Comment by benface** [comment]
@brandonkelly Could the same thing be done to the "Slug" field?

**Comment by brandonkelly** [comment]
@benface Yep, done.

**Comment by ghost** [comment]
Hi @brandonkelly  ,
I have a question for you.
I use 'site.php' to translation 'entry type names' for template, But why translated 'entry type names' for control panel? Am I reading document wrong or BUG? 
These things happen on craft 3.3.17.

**Comment by brandonkelly** [comment]
@DandeSuen It should work in the control panel as well. Please write into support@craftcms.com with *exactly* what you’re doing, and we can help determine why it’s not working.
